cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -591,7 +591,7 @@ func main() {
 	debugList := flag.Bool("debug-list", false, "do not start the indexserver, rather list the repositories owned by this indexserver then quit.")
 	debugIndex := flag.String("debug-index", "", "do not start the indexserver, rather index the repositories then quit.")
 
-	expGitIndex := flag.Bool("exp-git-index", os.Getenv("SRC_GIT_INDEX") != "", "use experimental indexing via shallow clones and zoekt-git-index")
+	expGitIndex := flag.Bool("exp-git-index", true, "use experimental indexing via shallow clones and zoekt-git-index")
```

Comment:
can you keep it so we can still disable via a env var? Just so we have easy flexibility if this causes issues at a customer. Right now adding a flag into the k8s environment is tricky (due to how we use entrypoint vs command in the docker container)

Reply:
Good idea. Now customers can set the env var `DISABLE_GIT_INDEX` to switch back to the old setting.